pos_margin/i18n/es.po

```diff
@@ -35,7 +35,8 @@ msgstr ""
 
 #. module: pos_margin
 #: model:ir.model,name:pos_margin.model_pos_order_line
-msgid "Lines of Point of Sale"
+#, fuzzy
```

Comment:
I think there is some errors in the translation file. there are a lot of "fuzzy".